subsys/net/lib/lwm2m/lwm2m_shell.c

```diff
@@ -466,6 +470,50 @@ static int cmd_unlock(const struct shell *sh, size_t argc, char **argv)
 	return 0;
 }
 
+static int cmd_cache(const struct shell *sh, size_t argc, char **argv)
+{
+	if (argc != 3) {
+		shell_error(sh, "wrong parameters\n");
+		return -EINVAL;
+	}
+
+	size_t plen = strlen(argv[1]) + 1;
+	char *path = k_malloc(plen);
+	if (!path) {
```

Comment:
Personally I prefer to keep all declarations at top of the code, it keeps the rest of the code more compact (no need for empty line then).

Reply:
Moved to beginning of function.